pkg/thing/interactors/list_things.go

```diff
@@ -10,10 +10,14 @@ func (i *ThingInteractor) List(authorization string) error {
 
 	things, err := i.thingProxy.List(authorization)
 	if err != nil {
+		sendErr := i.clientPublisher.SendDevicesList(things, err)
+		if sendErr != nil {
+			return fmt.Errorf("error getting list of things: %v, %w", sendErr, err)
+		}
 		return fmt.Errorf("error getting list of things: %w", err)
 	}
 
-	err = i.clientPublisher.SendDevicesList(things)
+	err = i.clientPublisher.SendDevicesList(things, nil)
```

Comment:
Please, prefer passing `err` instead of `nil` directly to increase function cohesion.

Reply:
@joaoaneto You're right! I gonna make some changes

Reply:
Done!